.github/workflows/cla-check.yaml

```diff
@@ -19,4 +19,4 @@ jobs:
           # use a shallow checkout.
           fetch-depth: 0
       - name: CLA check
-        run: ./tests/lib/cla_check.py
+        run: ./tests/lib/cla_check.py "${{ github.base_ref }}..${{ github.head_ref }}"
```

Comment:
This is failing because `github.base_ref` does not exist in the local git repo: only `github.head_ref` is valid.  I think this could be fixed by adding another step to the job just before this:

    - name: Fetching base ref ${{ github.base_ref }}
      run: git fetch origin ${{ github.base_ref }}:${{ github.base_ref }}

With that reference fetched, I believe the range you've specified should return the right set of revisions.

Reply:
That sounds plausible. I'll try pushing a patch with that additional step.